src/interface/cypress/e2e/accounts.spec.cy.ts

```diff
@@ -21,23 +21,6 @@ describe('Create an account with unmatching password', () => {
     cy.contains('Given passwords must match.');
     cy.get('button[type="submit"]').should('be.disabled');
   });
-
-  it('Attempts to create an account', () => {
```

Comment:
would it make sense to keep this test but omit it if env = prod? (so we can still have it if we want to run it locally)

Reply:
That's a good thought -- although now that I think about it...
Since these tests are just reading the baseUrl from the .env file on the test runner, and not the target server that's being tested, I think enforcing the actual server could be complex and misleading, (especially if we're using backend proxies).

Fwiw, my plan for the future was to have two suites of tests: a full set for local dev, and a set of smoke tests for live servers. So, maybe for the moment, I'll plan to keep this code in an archive, but I'll think about how we can reuse it later.